tests/functional/test_install_extras.py

```diff
@@ -14,7 +14,7 @@ def test_simple_extras_install_from_pypi(script):
         'install', 'Paste[openid]==1.7.5.1', expect_stderr=True,
     )
     initools_folder = script.site_packages / 'openid'
-    assert initools_folder in result.files_created, result.files_created
+    result.did_create(initools_folder)
```

Comment:
I think here we are providing the message as `result.files_created` instead of the default which is `str(result)`. So, I think it should be like
```suggestion
    result.did_create(initools_folder, result.files_created)
```

Reply:
No -- this information is included in `str(result)`.